.github/workflows/publish-common.yml

```diff
@@ -38,20 +40,31 @@ jobs:
       - name: Prepare environment
         uses: ./.github/actions/prepare-environment
 
+      - name: Lerna found changed packages
+        id: lerna
+        uses: ./.github/actions/lerna-changed
+
+      - name: Create "HAS_CHANGED" shortcut
+        run: echo "HAS_CHANGED=${{ steps.lerna.outputs.HAS_CHANGED }}" >> $GITHUB_ENV
+
       - name: Lerna bootstrap
+        if: ${{ fromJSON(env.HAS_CHANGED) }}
```

Comment:
[Note]: 

@Yeti-or @neretin-trike @kayman233 

Вот хотелось бы обсудить какой подход нам больше нравиться. 

Первый вариант, это у нас есть action в котором высчитывается результат `npx lerna changed -a` 
и потому это результат используется  как `if: ${{ fromJSON(env.HAS_CHANGED) }}` в самих workflow.

Собственно этот пример и описан в этих commit's.

Можно сделать комбинацию: у нас будет и action что бы на уровне конфигурации и запускать наши npm scripts в связке c `npx lerna changed -a`, что то типо такого

```json
{
"scripts": {
    "bootstrap": "npx lerna changed -a && npx lerna bootstrap",
    "bootstrap:since": "npm run bootstrap -- --since=$(git merge-base --fork-point origin/dev)",
}
}
```

Reply:
И тогда `npx lerna bootstrap` вообще не запуститься, потому что нет изменений. 

Опять же это справедливо для тех workflow где мы не используем флаг since. Например publish canary